### PR TITLE
Add support for a more developer-friendly environment and cleanup timbot functionality.

### PR DESCRIPTION
**Pull request by timflannagan**

### CODE REVIEW

File: src/timbot.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    `when lunch/lunch time`: i tell you when to eat lunch
+    `what lunch/what eat`: i tell you what to eat
+    `show webopoly standings`: show who is captalist king and who is poor
+    `help`: this
+'''
+
+
+# TODO: Change this to handle daylight savings
+def utc_to_est(utc_time):
+    return str((int(utc_time[:2]) - 5) % 24) + ':' + utc_time[-2:]
```
Comment (nathan-weinberg) [inline src/timbot.py:32]: Does this fix #64?

File: src/timbot.py [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+        responce = sc.api_call(
+            "files.upload",
+            title='sampletitle',
+            file=io.BytesIO(f.read())
+        )
+    fileinfo = responce['file']
+    sc.api_call("files.sharedPublicURL", file=fileinfo['id'])
+    image_url = fileinfo['permalink_public']
+    attachments = [{"title": title, "image_url": image_url}]
+    responce = sc.api_call("chat.postMessage", channel=channel_name, text=text, as_user=use_authenticated_user, attachments=attachments)
```
Comment (nathan-weinberg) [inline src/timbot.py]:
```suggestion
def uploadimage(path, title, text):
    with open(path, 'rb') as f:
        response = sc.api_call(
            "files.upload",
            title='sampletitle',
            file=io.BytesIO(f.read())
        )
    fileinfo = response['file']
    sc.api_call("files.sharedPublicURL", file=fileinfo['id'])
    image_url = fileinfo['permalink_public']
    attachments = [{"title": title, "image_url": image_url}]
    response = sc.api_call("chat.postMessage", channel=channel_name, text=text, as_user=use_authenticated_user, attachments=attachments)
```

**Review by nathan-weinberg — CHANGES_REQUESTED** [review]

File: src/timbot.py (hunk 1 above)
Comment (timflannagan) [reply src/timbot.py:32]: I didn't touch this function. You may have to look at the commits to see the changes I made.

File: src/timbot.py (hunk 2 above)
Comment (timflannagan) [reply src/timbot.py]: Updated

File: src/timbot.py (hunk 1 above)
Comment (nathan-weinberg) [reply src/timbot.py:32]: Since the file location changed, the diff shows the entire file as an addition

**Review by nathan-weinberg — CHANGES_REQUESTED** [review]
Overall LGTM, two small things:
1. Do we need to change the build check in GitHub to align with the makefile changes?
2. Please squash commits before merging

**Comment by timflannagan** [comment]
> Do we need to change the build check in GitHub to align with the makefile changes?

Nope, already checked.

**Review by nathan-weinberg — APPROVED** [review]

**Pull request merged**